<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EP-CP-102" name="termProduct.TermProductJourney" tests="13" failures="1" timestamp="7 Jul 2017 10:14:54 GMT" time="146.692" errors="0">
  <testcase name="GetLiveQuotes" time="69.140" classname="termProduct.TermProductJourney"/>
  <testcase name="TobaccoUser" time="1.125" classname="termProduct.TermProductJourney"/>
  <testcase name="LaunchWebsite" time="8.472" classname="termProduct.TermProductJourney"/>
  <testcase name="QuotesBuyNow" time="45.171" classname="termProduct.TermProductJourney">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at termProduct.TermProductJourney.CustomerPopup(TermProductJourney.java:309)
	at termProduct.TermProductJourney.QuotesBuyNow(TermProductJourney.java:393)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="City" time="2.085" classname="termProduct.TermProductJourney"/>
  <testcase name="TermDOB" time="3.491" classname="termProduct.TermProductJourney"/>
  <testcase name="EldestMember" time="1.137" classname="termProduct.TermProductJourney"/>
  <testcase name="CustomerMobno" time="0.213" classname="termProduct.TermProductJourney"/>
  <testcase name="CustomerPopup" time="8.483" classname="termProduct.TermProductJourney"/>
  <testcase name="FillLoginDetails" time="2.947" classname="termProduct.TermProductJourney"/>
  <testcase name="InvestAmount" time="3.336" classname="termProduct.TermProductJourney"/>
  <testcase name="CustomerEmail" time="0.348" classname="termProduct.TermProductJourney"/>
  <testcase name="CustomerProceed" time="0.744" classname="termProduct.TermProductJourney"/>
</testsuite> <!-- termProduct.TermProductJourney -->
